biomage/stage/stage.py

```diff
@@ -285,21 +637,24 @@ def stage(token, org, deployments):
     if not answers["create"]:
         exit(1)
```

Comment:
what is this doing?

Reply:
Hmmm.. I believe that's to check if the key doesn't exist. Don't think it's necessary to check for that though.